scss/civicrm/dashboard/pages/_view-dashboard.scss

```diff
@@ -16,6 +18,10 @@
 
     .widget-content {
       padding: 0;
+
+      .status.no-popup {
+        border: 0;
```

Comment:
did you consult with Jon about this decision?

Reply:
Yep